examples/sentence-transformers-training/nli/README.md

```diff
@@ -46,6 +48,28 @@ For multi-card training you can use the script of [gaudi_spawn.py](https://githu
 HABANA_VISIBLE_MODULES="2,3" python ../../gaudi_spawn.py --use_deepspeed --world_size 2 training_nli.py bert-base-uncased
 ```
 
+
+# intfloat/e5-mistral-7b-instruct Model
```

Comment:
@nngokhale the headers and sub-headers do not make sense! you have `Single-card Training` under `General Models` and later have `intfloat/e5-mistral-7b-instruct Model` at the same level as `General Models` 
Please review the REAME and make reasonable adjustment!

Reply:
Done